handbook/engineering/career-development/framework.md

```diff
@@ -30,6 +33,8 @@ When your manager can make the case that you’ve had at least one quarter of hi
 Promotions from one level to another are considered in our quarterly [talent review](talent-review-process.md).
 An in-band compensation increase (while staying at the same level) can happen at any time, in recognition of exceeding expectations in your current level without having yet met the expectations of the next level.
 
+You can learn more [here](talent-review-process.md#considerations-for-promotion).
```

Comment:
Generally, links should not say "here" or "click here", but the content of the link should be descriptive in itself: https://webaccess.berkeley.edu/ask-pecan/click-here

A big reason for this is accessibility, screen readers allow to jump to links in a document/paragraph/section and will announce the content of the link, but if it only says "here", the user can't know where it leads.

Better would be: `To learn more, read our [talent review process](...).`

Will change this on `main` but just wanted to share this tip for the future :)

Reply:
Thanks!